com.unity.multiplayer.mlapi/Runtime/Core/NetworkBehaviour.cs

```diff
@@ -25,6 +25,7 @@ namespace MLAPI
     /// </summary>
     public abstract class NetworkBehaviour : MonoBehaviour
     {
+#pragma warning disable IDE1006
```

Comment:
disabling and restoring naming rules (Code Rule ID: `IDE1006 (Naming rule violation)`) in certain blocks specifically if the intention is clear and non-standard naming is required, such as `__NExec`, `__ntable` and other RPC ILPP infrastructure.